### ISSUE TITLE
app-accessibility/onboard-1.4.1 - build failure

### ISSUE THREAD

**Issue by leonchik1976** [pimvullers/elementary#268]
Onboard/osk/osk_audio.c:70:5: warning: ‘gdk_screen_get_number’ is deprecated [-Wdeprecated-declarations]
   70 |     nr = gdk_screen_get_number(screen);
      |     ^~
In file included from /usr/include/gtk-3.0/gdk/gdkapplaunchcontext.h:31,
                 from /usr/include/gtk-3.0/gdk/gdk.h:32,
                 from /usr/include/gtk-3.0/gdk/gdkx.h:32,
                 from Onboard/osk/osk_audio.c:23:
/usr/include/gtk-3.0/gdk/gdkscreen.h:56:14: note: declared here
   56 | gint         gdk_screen_get_number            (GdkScreen   *screen);
      |              ^~~~~~~~~~~~~~~~~~~~~
Onboard/osk/osk_audio.c: In function ‘osk_audio_play’:
Onboard/osk/osk_audio.c:101:5: warning: ‘gdk_screen_get_width’ is deprecated [-Wdeprecated-declarations]
  101 |     sw = gdk_screen_get_width(screen);
      |     ^~
/usr/include/gtk-3.0/gdk/gdkscreen.h:58:14: note: declared here
   58 | gint         gdk_screen_get_width             (GdkScreen   *screen);
      |              ^~~~~~~~~~~~~~~~~~~~
Onboard/osk/osk_audio.c:102:5: warning: ‘gdk_screen_get_height’ is deprecated [-Wdeprecated-declarations]
  102 |     sh = gdk_screen_get_height(screen);
      |     ^~
/usr/include/gtk-3.0/gdk/gdkscreen.h:60:14: note: declared here
   60 | gint         gdk_screen_get_height            (GdkScreen   *screen);
      |              ^~~~~~~~~~~~~~~~~~~~~
error: command '/usr/lib/ccache/bin/x86_64-pc-linux-gnu-gcc' failed with exit code 1
found dconf version 0.40.0
 * ERROR: app-accessibility/onboard-1.4.1::elementary failed (compile phase):
 *   Wheel build failed
 *
 * Call stack:
 *     ebuild.sh, line  136:  Called src_compile
 *   environment, line 3850:  Called distutils-r1_src_compile
 *   environment, line 1707:  Called _distutils-r1_run_foreach_impl 'distutils-r1_python_compile'
 *   environment, line  607:  Called python_foreach_impl 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 3497:  Called multibuild_foreach_variant '_python_multibuild_wrapper' 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 3040:  Called _multibuild_run '_python_multibuild_wrapper' 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 3038:  Called _python_multibuild_wrapper 'distutils-r1_run_phase' 'distutils-r1_python_compile'
 *   environment, line 1045:  Called distutils-r1_run_phase 'distutils-r1_python_compile'
 *   environment, line 1689:  Called distutils-r1_python_compile
 *   environment, line 1584:  Called distutils_pep517_install '/var/tmp/portage/app-accessibility/onboard-1.4.1/work/onboard-1.4.1-python3_11/install'
 *   environment, line 2050:  Called die
 * The specific snippet of code:
 *       local wheel=$("${cmd[@]}" 3>&1 1>&2 || die "Wheel build failed");

[build.log](https://github.com/user-attachments/files/22134095/build.log)

[emerge_info.txt](https://github.com/user-attachments/files/22134096/emerge_info.txt)
